.circleci/config.yml

```diff
@@ -27,22 +22,23 @@ jobs:
       # Download and cache dependencies
       - restore_cache:
           keys:
-            - v1-dependencies-{{ checksum "pom.xml" }}
+            - v1-dependencies-{{ checksum "pom.xml" }}-{{ checksum "front-end/package.json" }}
             # fallback to using the latest cache if no exact match is found
             - v1-dependencies-
 
       - run: mvn dependency:go-offline
+      - run: cd front-end && npm install
 
       - save_cache:
           paths:
             - ~/.m2
-          key: v1-dependencies-{{ checksum "pom.xml" }}
+          key: v1-dependencies-{{ checksum "pom.xml" }}-{{ checksum "front-end/package.json" }}
 
-      - run: mvn test
+      - run: mvn package
+      - run: cd front-end && npm test
 
-      - store_test_results: # uploads the test metadata from the `target/test-results` directory so that it can show up in the CircleCI dashboard.
-          # Upload test results for display in Test Summary: https://circleci.com/docs/2.0/collect-test-data/
+      - store_test_results:
           path: target/test-results
```

Comment:
You mentioned yesterday that this wasn't working, is it now? if is not working should we just remove or fix it? (just curious)

Reply:
its done